test/credentials.yml

```diff
@@ -11,9 +11,11 @@ usps:
   login: <%= ENV['ACTIVESHIPPING_USPS_LOGIN'] %>
 
 ups:
-  key: <%= ENV['ACTIVESHIPPING_UPS_LOGIN'] %>
-  login: <%= ENV['ACTIVESHIPPING_UPS_KEY'] %>
-  password: <%= ENV['ACTIVESHIPPING_UPS_PASSWORD'] %>
+  key: 5CD7DE7593CBF516
+  login: apicasperuser
+  password: T30roFStkUyn
+  origin_account: W5A760
+  origin_name: Casper
```

Comment:
This was me being stupid. I went ahead and changed my password and deactivated that key.